package.json

```diff
@@ -49,9 +49,9 @@
     "@map-colonies/schemas": "^1.12.2",
     "@map-colonies/telemetry": "^10.0.1",
     "@opentelemetry/api": "^1.9.0",
-    "@prisma/adapter-pg": "^6.11.0",
-    "@prisma/client": "^6.11.0",
-    "@prisma/instrumentation": "^6.12.0",
+    "@prisma/adapter-pg": "^6.13.0",
+    "@prisma/client": "^6.13.0",
+    "@prisma/instrumentation": "^6.13.0",
```

Comment:
13 is a cursed number, please fix

Reply:
![Uploading image.png…]()

Reply:
<img width="543" height="737" alt="image" src="https://github.com/user-attachments/assets/74d47cae-3ff8-42f2-8d1a-85310d258827" />